From 893acdfd24329198ab8b930c1b124e64b0c9b3dc Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Wed, 23 Jan 2013 16:35:37 +0800
Subject: [PATCH 155/310] ENGR00241418 Remove telephony feature for data
 connection only device

Cannot have telephony and gsm feature ifconfig_voice_capable be false
Pass the below cts cases with this change:
android.app.cts.SystemFeaturesTest#testTelephonyFeatures
android.telephony.gsm.cts.SmsManagerTest#testSendMessages
android.telephony.cts.SmsManagerTest#testSendMessages

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arm2_6dq/required_hardware.xml     | 2 --
 arm2_6sl/required_hardware.xml     | 2 --
 evk_6sl/required_hardware.xml      | 2 --
 imx53_smd/required_hardware.xml    | 2 --
 imx6/hdmidongle_6dq.mk             | 1 -
 imx6/sabresd_6dq.mk                | 1 -
 sabreauto_6q/required_hardware.xml | 2 --
 7 files changed, 12 deletions(-)

diff --git a/arm2_6dq/required_hardware.xml b/arm2_6dq/required_hardware.xml
index d2c215e..8b0830d 100644
--- a/arm2_6dq/required_hardware.xml
+++ b/arm2_6dq/required_hardware.xml
@@ -25,8 +25,6 @@
  -->
     <feature name="android.hardware.location" />
     <feature name="android.hardware.location.network" />
-    <feature name="android.hardware.telephony" />
-    <feature name="android.hardware.telephony.gsm" />
     <feature name="android.hardware.wifi" />
     <feature name="android.hardware.touchscreen" />
     <feature name="android.hardware.touchscreen.multitouch" />
diff --git a/arm2_6sl/required_hardware.xml b/arm2_6sl/required_hardware.xml
index d2c215e..8b0830d 100644
--- a/arm2_6sl/required_hardware.xml
+++ b/arm2_6sl/required_hardware.xml
@@ -25,8 +25,6 @@
  -->
     <feature name="android.hardware.location" />
     <feature name="android.hardware.location.network" />
-    <feature name="android.hardware.telephony" />
-    <feature name="android.hardware.telephony.gsm" />
     <feature name="android.hardware.wifi" />
     <feature name="android.hardware.touchscreen" />
     <feature name="android.hardware.touchscreen.multitouch" />
diff --git a/evk_6sl/required_hardware.xml b/evk_6sl/required_hardware.xml
index 4ccf78e..1da8cfe 100644
--- a/evk_6sl/required_hardware.xml
+++ b/evk_6sl/required_hardware.xml
@@ -25,8 +25,6 @@
  -->
     <feature name="android.hardware.location" />
     <feature name="android.hardware.location.network" />
-    <feature name="android.hardware.telephony" />
-    <feature name="android.hardware.telephony.gsm" />
     <feature name="android.hardware.wifi" />
     <feature name="android.hardware.faketouch" />
     <feature name="android.hardware.touchscreen" />
diff --git a/imx53_smd/required_hardware.xml b/imx53_smd/required_hardware.xml
index f5e8d42..76341d1 100644
--- a/imx53_smd/required_hardware.xml
+++ b/imx53_smd/required_hardware.xml
@@ -23,8 +23,6 @@
     <feature name="android.hardware.location" />
     <feature name="android.hardware.location.network" />
     <feature name="android.hardware.location.gps" />
-    <feature name="android.hardware.telephony" />
-    <feature name="android.hardware.telephony.gsm" />
     <feature name="android.hardware.wifi" />
     <feature name="android.hardware.wifi.direct" />
     <feature name="android.hardware.touchscreen" />
diff --git a/imx6/hdmidongle_6dq.mk b/imx6/hdmidongle_6dq.mk
index 80f5178..613b584 100644
--- a/imx6/hdmidongle_6dq.mk
+++ b/imx6/hdmidongle_6dq.mk
@@ -28,7 +28,6 @@ PRODUCT_COPY_FILES += \
 	frameworks/native/data/etc/android.hardware.wifi.direct.xml:system/etc/permissions/android.hardware.wifi.direct.xml \
 	frameworks/native/data/etc/android.hardware.sensor.light.xml:system/etc/permissions/android.hardware.sensor.light.xml \
 	frameworks/native/data/etc/android.hardware.sensor.accelerometer.xml:system/etc/permissions/android.hardware.sensor.accelerometer.xml \
-	frameworks/native/data/etc/android.hardware.telephony.gsm.xml:system/etc/permissions/android.hardware.telephony.gsm.xml \
 	frameworks/native/data/etc/android.hardware.faketouch.xml:system/etc/permissions/android.hardware.faketouch.xml \
 	frameworks/native/data/etc/android.hardware.sensor.barometer.xml:system/etc/permissions/android.hardware.sensor.barometer.xml \
 	frameworks/native/data/etc/android.hardware.camera.front.xml:system/etc/permissions/android.hardware.camera.front.xml \
diff --git a/imx6/sabresd_6dq.mk b/imx6/sabresd_6dq.mk
index 70c36f2..effa714 100644
--- a/imx6/sabresd_6dq.mk
+++ b/imx6/sabresd_6dq.mk
@@ -29,7 +29,6 @@ PRODUCT_COPY_FILES += \
 	frameworks/native/data/etc/android.hardware.wifi.direct.xml:system/etc/permissions/android.hardware.wifi.direct.xml \
 	frameworks/native/data/etc/android.hardware.sensor.light.xml:system/etc/permissions/android.hardware.sensor.light.xml \
 	frameworks/native/data/etc/android.hardware.sensor.accelerometer.xml:system/etc/permissions/android.hardware.sensor.accelerometer.xml \
-	frameworks/native/data/etc/android.hardware.telephony.gsm.xml:system/etc/permissions/android.hardware.telephony.gsm.xml \
 	frameworks/native/data/etc/android.hardware.faketouch.xml:system/etc/permissions/android.hardware.faketouch.xml \
 	frameworks/native/data/etc/android.hardware.sensor.barometer.xml:system/etc/permissions/android.hardware.sensor.barometer.xml \
 	frameworks/native/data/etc/android.hardware.camera.front.xml:system/etc/permissions/android.hardware.camera.front.xml \
diff --git a/sabreauto_6q/required_hardware.xml b/sabreauto_6q/required_hardware.xml
index d09af05..e2d8a05 100644
--- a/sabreauto_6q/required_hardware.xml
+++ b/sabreauto_6q/required_hardware.xml
@@ -22,8 +22,6 @@
     <feature name="android.hardware.camera.flash" />
     <feature name="android.hardware.location" />
     <feature name="android.hardware.location.network" />
-    <feature name="android.hardware.telephony" />
-    <feature name="android.hardware.telephony.gsm" />
     <feature name="android.hardware.wifi" />
     <feature name="android.hardware.touchscreen" />
     <feature name="android.hardware.touchscreen.multitouch" />
-- 
1.8.0

